frontend/components/Projectcard.js

```diff
@@ -1,44 +1,16 @@
 import style from '../styles/Projectcard.module.css'
 
-const ProjectCard = () => {
-  return (
-    <div className={style.card}>
-      <div className={style.container}>
-        <img />
-      </div>
-      <div>
-        <h5><strong>Name</strong></h5>
-        <h6>Title</h6>
-      </div>
-    </div>
-  )
-}
-
-const ProjectCardPlaceholder = ({ project }) => {
+const ProjectCard = ({ project }) => {
   return (
     <article className={style.card}>
-      <div className={style.image}>
-        <svg
-          className='bd-placeholder-img bd-placeholder-img-lg'
-          width='320'
-          height='180'
-          xmlns='http://www.w3.org/2000/svg'
-          role='img'
-          preserveAspectRatio='xMidYMid slice'
-          focusable='false'
-        >
-          <title>Placeholder</title>
-          <rect width='100%' height='100%' fill='grey' />
-          <text x='50%' y='50%' fill='white' dy='.3em' dominantBaseline='middle' textAnchor='middle'>{project.Name}</text>
-        </svg>
-      </div>
+      <iframe width='320' height='180' src={project.projectLink} allowFullScreen> </iframe>
```

Comment:
Bootstrap suggests using an descendant class '.embed-responsive-item' for videos so that their aspect ratio/size changes based on the size of the parent div/element housing the embedded video.

This isn't necessary right now since our project cards are of fixed size, but may need to be changed later on anyways.

[https://getbootstrap.com/docs/4.0/utilities/embed/](url)
[https://stackoverflow.com/questions/40332537/when-to-use-the-bootstrap-class-attribute-value-embed-responsive-item](url)